### PR TITLE
Resolve metainfo linter warnings

### PR DESCRIPTION
**Pull request by carstene1ns**
While adding 0.8 to flathub[^1] their linker gave 2 warnings:
"appstream-summary-too-long"[^2]
"appstream-screenshot-missing-caption"[^3]

This resolves both of them, but I am still unsure about the short[sic!] summary...
The now enabled URLs are untested, I just think this must be supported after 2 years. 🏂🏼

[^1]: https://github.com/flathub/flathub/pull/5088
[^2]: https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines/#not-too-long-1
[^3]: https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#screenshots

### CODE REVIEW

**Comment by carstene1ns** [comment]
This now also fixes the liblcf build.

**Comment by Ghabry** [comment]
well I have no experience with flatpak so feel free to merge this when you consider it done.

**Comment by carstene1ns** [comment]
Just waiting to clear the nits found by flathub maintainers before amending here.

**Review by fdelapena — APPROVED** [review]

**Pull request merged**
